code/modules/research/designs/stock_parts.dm

```diff
@@ -102,7 +102,6 @@
 	build_path = /obj/item/bluespace_crystal/artificial
 	sort_string = "VAAAF"
 
-
```

Comment:
This is a ghost file change but I'm not sure if those are easily undone.

Reply:
Line ending fix, my git bash commit setted up on Unix line ending, not like stupid github desktop 🤷‍♂